.github/workflows/release.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       - name: "4. Build and publish docker to docker hub"
         uses: elgohr/Publish-Docker-Github-Action@2.14
         with:
-          name: unit8/u8timeseries
+          name: unit8/darts
```

Comment:
@endrjuskr do we need to create something new on Dockerhub for that to work?

Reply:
yes I need, will take care of it

Reply:
done :)